### DISCUSSION TITLE
Use ory/keto to authorize OpenStack and Kubernetes

### DISCUSSION THREAD

**Discussion by gujun4990** [ory/keto#609] [Ideas]
Hi,
Currently, we want to realize a unified authorization system to authorize OpenStack and Kubernetes. After researching the ory/keto, we found that it is an amazing project. But we found that there's no a good way to access to OpenStack and Kubernetes. So we want to know whether there are some methods to take over OpenStack and Kubernetes's authorization service using ory/keto.

Thanks!

**Comment by aeneasr** [comment]
We haven't investigated that yet - especially because we do not use OpenStack. If you figure out how to include it, I think many people would be interested to know :)

**Reply by gujun4990** [reply]
OK. Thank you!
